app/models/illumina_htp/mx_tube_purpose.rb

```diff
@@ -4,38 +4,8 @@
 # pipeline. These tubes represent the cleaned-up normalized libraries at the end
 # of the process that can pass directly into a {SequencingPipeline}.
 # State changes on these tubes will automatically update the requests into the tubes
-# @note Most current activity is on subclasses of this purpose, especially IlluminaHtp::MxTubeNoQcPurpose
-#       As of 2019-10-01 only used directly by 'Lib Pool Norm' and 'Lib Pool SS-XP-Norm' which haven't been
-#       used since 2017-04-28 14:16:03 +0100
 class IlluminaHtp::MxTubePurpose < Tube::Purpose
-  # Called via Tube#transition_to
-  # Updates the state of tube to state
```

Comment:
More state change extraction. I had hoped to be able to eliminate a few of the tube state changers, and still think it should be possible, but they weren't quite as similar as I'd have liked. So keeping risk low for the time being.